application/views/debug/index.php

```diff
@@ -461,3 +466,7 @@
     </div>
 
 </div>
+
+<script>
+    var local_branch = '<?php echo $branch; ?>';
```

Comment:
this one fails / is empty when using a version which wasn't installed via git.

Reply:
catched